src/vivarium_gates_bep/globals.py

```diff
@@ -21,6 +21,114 @@
 ]
 
 
+###################
+# Model Constants #
+###################
+# This is a collection of constants used in data generation or as
+# invariant model parameters.
+
+def twenty_percent_of_mean_variance(mean):
+    ninety_five_percent_spread = .2 * mean
+    std_dev = ninety_five_percent_spread / (2 * 1.96)
+    return std_dev ** 2
+
+
+def confidence_interval_variance(upper, lower):
+    ninety_five_percent_spread = (upper - lower) / 2
+    std_dev = ninety_five_percent_spread / (2 * 1.96)
+    return std_dev ** 2
+
+
+MALNOURISHED_MOTHERS_PROPORTION_MEAN = {
```

Comment:
may be worth it to comment where these numbers came from, they look like magic. same below

Reply:
I guess there are a ton of numbers in this PR...

Reply:
Yeah, I'd like to put the concept model in the thing when we archive for that sort of sourcing.